local-cluster/tests/local_cluster.rs

```diff
@@ -478,6 +478,64 @@ fn test_mainnet_beta_cluster_type() {
     }
 }
 
+#[test]
+#[serial]
+#[ignore = "only for replay invalidator"]
```

Comment:
i don't think we really want these to be public?

Reply:
The ignore message or the test itself? Test doesn't add any real value w/o being hooked up to secret stuff (just 2 validators forming a cluster for 30 seconds w/o any traffic)

Reply:
Removed the test